run-bahmni.sh

```diff
@@ -0,0 +1,174 @@
+
+    if [ "${DOCKER_SERVER_VERSION_MAJOR}" -ge 20 ] && \
+    [ "${DOCKER_SERVER_VERSION_MINOR}" -ge 10 ]  && \
+    [ "${DOCKER_SERVER_VERSION_BUILD}" -ge 13 ]; then
+        echo 'Docker version >= 20.10.13, using Docker Compose V2'
+    else
+        echo 'Docker versions < 20.10.13 are not supported' >&2 
+        exit 1
+    fi
+
+    # Check the version of Docker Compose
+    if ! [ -x "$(command -v docker compose version)" ]; then
+    echo 'Error: docker compose is not installed. Please install docker compose.' >&2
+    exit 1
+    fi
+    version=$(docker compose version)
+    echo "Docker Compose version: $version"
+    echo "---"
+}
+
+function checkIfDirectoryIsCorrect {
+    # Current subdirectory
+    current_subdir=$(basename $(pwd))
+    echo "$current_subdir"
+
+    if [ "$current_subdir" == "bahmni-lite" ] || [ "$current_subdir" == "bahmni-standard" ] ; then
+        return
+    else
+        echo "Error: This script should be run from either 'bahmni-lite' or 'bahmni-standard' subfolder. Please cd to the appropriate sub-folder and then execute the run-bahmni.sh command."
+        exit 1
+    fi
+}
+
+function start {
+    echo "Executing command: 'docker compose up -d'"
+    docker compose up -d
+}
+
+
+function stop {
+    echo "Executing command: 'docker compose down'"
+    docker compose down
+}
+
+function sshIntoService {
+    echo "Listing the running services..."
+    docker compose ps
+
+    echo "Enter the SERVICE name which you wish to ssh into:"
+    read serviceName
+    
+    docker compose exec $serviceName /bin/sh
+}
+
+function showLogsOfService {
+    echo "Listing the running services..."
+    docker compose ps
+
+    echo "Enter the SERVICE name whose logs you wish to see:"
+    read serviceName
+    
+    docker compose logs $serviceName -f
+}
+
+
+function showOpenMRSlogs {
+    echo "Opening OpenMRS Logs..."
+    docker compose logs openmrs -f 
+}
+
+function startMart {
+    echo "Starting services with profile 'bahmni-mart'..."
+    docker compose --profile bahmni-mart up -d
+}
+
+function pullLatestImages {
+    echo "Pulling all latest images..."
+    docker compose pull
+}
+
+function showStatus {
+    echo "Listing status of running services with command: 'docker ps'"
+    docker ps
+}
+
+
+# Function to prompt the user for a "Yes" or "No" answer
+confirm() {
+    read -p "$1 [y/n]: " response
+    case $response in
+        [yY][eE][sS]|[yY])
+            return 0
+            ;;
+        [nN][oO]|[nN])
+            return 1
+            ;;
+        *)
+            echo "Invalid input"
+            return 1
+    esac
+}
+
+
+function resetAndEraseALLVolumes {
+  if confirm "Are you sure you want to DELETE all Data and Volumes in docker on this MACHINE?? Please say Yes/No to proceed if you are absolutely sure. This will delete even non-Bahmni volumes used by Docker!"; then
+    echo "Proceeding with a DELETE.... "
+    
+    echo "1. Stopping all services.."
+    docker compose down
```

Comment:
We can use -v flag in  this command to remove volumes created by docker compose.

Reply:
Let me try -v. The problem is, that if a service is not running, its volumes will not be deleted. Would want to delete all BAHMNI related volumes.

Reply:
Done. Please see.